firefox/package.json

```diff
@@ -6,7 +6,7 @@
 	"author": "honestbleeps",
 	"homepage": "http://redditenhancementsuite.com/",
 	"license": "GPL-3.0",
-	"version": "{{prop?version!../package.json}}",
+	"version": "{{to-string!./getVersion}}",
```

Comment:
Note: `jpm` warns about the version because it's not valid semver, despite the fact that AMO _doesn't even use_ semver!

For reference, [here are the docs](https://developer.mozilla.org/en-US/Add-ons/AMO/Policy/Maintenance) suggesting to use `1.1alpha1`, which isn't valid (`1.1.0-alpha.1` would be), and a [published extension](https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/#beta-channel) with version number `1.7.7b15`.